examples/agent-examples.yaml

```diff
@@ -0,0 +1,76 @@
+# Example: Agents and Agent Relationships
+
+'@context':
+  - https://git.io/vf-examples-jsonld-context
+  - alice: https://alice.example/
+    bob: https://bob.example/
+    fablab: https://fablab.example/
+    coop: https://coop.example/
+
+'@graph':
+
+  # Agents
+
+  - '@id': alice:
+    '@type': foaf:Person
+    name: 'Alice'
+    image: 'https://alice.example/avatar.png'
+    url: 'https://alice.example'
```

Comment:
can we remove this `url` not to cause confusion on how does it differ from `@id`?